packages/atoms/src/Icon/Icon.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {
+    Canvas,
+    ImageSVG,
+    useSVG,
+    Group,
+    Paint,
+    BlendColor,
+    usePaintRef,
+} from '@shopify/react-native-skia';
+import { IconType, iconTypes } from './iconTypes';
+import { useNativeStyles } from '@trezor/styles';
+import { Color } from '@trezor/theme';
+
+const iconSizes = {
+    small: 16,
+    big: 24,
+} as const;
+
+export type IconSize = keyof typeof iconSizes;
+
+export type IconProps = {
+    type: IconType;
+    size?: IconSize;
+    color?: Color;
+};
+
+export const Icon = ({ type, size = 'big', color = 'black' }: IconProps) => {
```

Comment:
Shouldn't the default color be `gray800`?

Reply:
probably yes, good catch

Reply:
I am probably going to keep it black because there is probably nothing like default from what I can see in Figma. There is lot of usages of black and shades of gray, so black would be probably best because you can easily recognize you need to change it